how-to/workspace-platform-starter/docs/how-to-add-a-service.md

```diff
@@ -0,0 +1,66 @@
+> **_:information_source: OpenFin Workspace:_** [OpenFin Workspace](https://www.openfin.co/workspace/) is a commercial product and this repo is for evaluation purposes (See [LICENSE.MD](../LICENSE.MD)). Use of the OpenFin Container and OpenFin Workspace components is only granted pursuant to a license from OpenFin (see [manifest](../public/manifest.fin.json)). Please [**contact us**](https://www.openfin.co/workspace/poc/) if you would like to request a developer evaluation key or to discuss a production license.
+
+[<- Back to Table Of Contents](../README.md)
+
+# How To Add A Service
+
+First a definition of what we mean by service:
+
+> A service is something that is not visible, automatically starts up when the platform starts, runs until the platform closes and executes some custom code created by a developer.
+
+There are two approaches to achieving the following:
+
+- Create a lifecycle module (see [how to use lifecycle events](./how-to-use-lifecycle-events.md))
+- Create a headless app and add it to your app directory configured to autostart (see [how to add an app](./how-to-define-apps.md))
+
+This document will describe the two options in more detail and the reasons why you might pick one over the other.
+
+## Lifecycle Module
+
+A lifecycle module is a JavaScript module that is built (see [how to use lifecycle events](./how-to-use-lifecycle-events.md)) by a platform developer or a content developer. A platform owner can then register the module through configuration so that it loads up and executed at the specified lifecycle event (e.g. after bootstrapping).
```

Comment:
Needs an is -- that it loads up and is executed...

Reply:
good catch